.github/workflows/unit.yml

```diff
@@ -44,7 +44,7 @@ jobs:
         pip install protobuf==3.7.1
 
     - run: yarn install --frozen-lockfile --network-timeout 1000000
-    - run: yarn build-report
```

Comment:
What were we missing out on before this change?

Reply:
> What were we missing out on before this change?

tests under `clients/test/extension/` needed `yarn build-extension` and `lightrider-entry-test.js` needed `yarn build-lr` and with `yarn build-report` already needed, `build-all` just seemed easier :)